.storybook/fs.js

```diff
@@ -0,0 +1,17 @@
+// WMH - Why am I using Filer instead of BrowserFS? IDK I just learned about Filer
+// recently and it's 1/2 the KB of BrowserFS so I thought I'd try it.
+import Filer from 'filer';
+const fs = new Filer.FileSystem();
+// Right now, only exporting members needed by @stoplight/graph
+// Filer.js doesn't support passing optiosn to readdir
+export const readdir = (path, opts, callback) => {
+  if (!callback) {
+    return fs.readdir.call(fs, path, opts);
+  } else {
+    return fs.readdir.call(fs, path, callback);
```

Comment:
Re-reading this — isn't in this case just ok to do `fs.readdir(path, opts)` or `fs.readdir(path, callback)`?

Reply:
Possibly, but I've run into cases where functions that I thought were pure functions turned out to be methods that relied implicitly on `this`.